colosseum/validators.py

```diff
@@ -0,0 +1,101 @@
+from . import parser
+from . import units
+
+
+def _numeric_validator(value, numeric_type, min_value, max_value):
+    error_msg = ''
+    try:
+        value = numeric_type(value)
+    except (ValueError, TypeError):
+        error_msg = "Cannot coerce {value} to {numeric_type}".format(
+            value=value, numeric_type=numeric_type.__name__)
+
+    if not error_msg and min_value is not None and value < min_value:
+        error_msg = 'Value {value} bellow minimum value {min_value}'.format(
+            value, min_value)
+
+    if not error_msg and max_value is not None and value > max_value:
+        error_msg = 'Value {value} above maximum value {max_value}'.format(
+            value, max_value)
+
+    return error_msg, value
+
+
+def is_number(value, min_value=None, max_value=None):
+    """
+    Validate that value is a valid float.
+
+    If min_value or max_value are provided, range checks are performed.
+    """
+
+    def validator(value):
+        return _numeric_validator(value, numeric_type=float, min_value=min_value, max_value=max_value)
+
+    if min_value is None and max_value is None:
+        return validator(value)
+    else:
+        return validator
```

Comment:
Are you sure this works as intended for the min/max-provided case? There's no test case for validating min/max (that I can see) - but to my reading, you won't be able to invoke `is_number(min_value=0)` and get a 0-min validating function returned.

Reply:
Yes, I need to add test, but wanted some feedback here since the idea is to be able to use the validator as 

```python

# No options for range
SOMETHING = Choices(AUTO, validators=[is_integer])

# Since it was not called with options, I need to check and return the actual validation result

# With range checks
SOMETHING = Choices(AUTO, validators=[is_integer(min_value=10, max_value=40])  

# Since it was called with options, I need to return a callable
```

Reply:
Ohhh duhh yes hahaha, ignore the last comment, yes I will fix this 😆